### DISCUSSION TITLE
When will MySQL be supported

### DISCUSSION THREAD

**Discussion by rydeanvi** [AntlerVC/firetable#1051] [Support / Q&A] [answered]
When will MySQL be supported?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @rydeanvi, we're currently working on the support for PostgreSQL with Rowy. PostgreSQL is considered to be an advanced version of MySQL. Therefore, it would be able to support the different functions of SQL, like the foreign keys, user-defined data types, and functions just as in MySQL.
